x-pack/plugins/reporting/server/browsers/phantom/node-phantom-simple/bridge.js

```diff
@@ -0,0 +1,260 @@
+/* eslint-disable */
+
+/*global phantom*/
+
+var webpage     = require('webpage');
```

Comment:
Curious what this file is (and the following file, too). They don't appear to be our own code? I only scanned them, didn't review thoroughly. But if these are being dropped in from some other project / source, it might be nice to comment them as such.

Reply:
They're from `@elastic/node-phantom-simple`. PhantomJS will be going away soon enough, so the need to make this readable is minimal.

Reply:
Make sure that we properly vendor and annotate with the necessary license info. @epixa might have opinions here.